var/spack/repos/builtin/packages/py-pillow/package.py

```diff
@@ -85,22 +86,26 @@ def patch(self):
                      'library_dirs = {0}'.format(library_dirs), string=True)
         setup.filter('include_dirs = []',
                      'include_dirs = {0}'.format(include_dirs), string=True)
-
-    def build_ext_args(self, spec, prefix):
-        def variant_to_flag(variant):
-            able = 'enable' if '+' + variant in spec else 'disable'
-            return '--{0}-{1}'.format(able, variant)
-
-        args = ['--enable-zlib', '--enable-jpeg']
-
-        variants = ['tiff', 'freetype', 'lcms', 'webp', 'webpmux', 'jpeg2000']
-        args.extend(list(map(variant_to_flag, variants)))
-
-        # Spack does not (yet) support these modes of building
-        args.append('--disable-imagequant')
-
-        args.append('--rpath={0}'.format(':'.join(self.rpath)))
-        return args
+        self._pillow_setup()
+
+    def _pillow_setup(self):
```

Comment:
Let's just do this directly in `patch` instead of creating a new function.

Reply:
ok